### PR TITLE
ui: tests for test section of overview pane

### PR DESCRIPTION
**Pull request by maiamcc**
Hello @nicks, @hyu,

Please review the following commits I made in branch maiamcc/test-section-tests:

5c6b5eda71a15ca774cf2445b201cdaa0b03b3b0 (2021-01-15 12:29:33 -0500)
ui: tests for test section of overview pane

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by maiamcc** [comment]
Oh wait let me make the error reporting a little better

**Comment by maiamcc** [comment]
...never mind, there's no clean way to do this, this level of verbosity in test output is fine--a user can go back through the stack trace to find where things went wrong if they need to :-/

File: web/src/OverviewPane.tsx [hunk 1]
```diff
@@ -40,31 +40,40 @@ let ServicesContainer = styled.div`
...
 `
 
-function PinnedServicesDivider() {
-  return (
-    <ServicesDividerRoot>
-      <GridDividerPinSvg style={{ marginLeft: "28px" }} />
-      <ServicesLabel>Pinned Resources</ServicesLabel>
-    </ServicesDividerRoot>
-  )
+function PinnedResources(items: OverviewItem[]) {
```
Comment (nicks) [inline web/src/OverviewPane.tsx:43]: can you make these React components that return React.Fragments?

File: web/src/OverviewPane.tsx [hunk 2]
```diff
@@ -40,31 +40,40 @@ let ServicesContainer = styled.div`
...
+  return items?.length ? (
+    <div className={"resources-container pinned"}>
```
Comment (nicks) [inline web/src/OverviewPane.tsx:45]: Let's get rid of all the CSS class names in this PR. None of them are used for styling. Once this is a React.Fragment, you should be able to use the react component itself as a selector.

**Comment by maiamcc** [comment]
thanks for the tip -- this should be better!

File: web/src/OverviewPane.tsx [hunk 3]
```diff
@@ -40,31 +40,44 @@ let ServicesContainer = styled.div`
...
-  return (
-    <ServicesDividerRoot>
-      <GridDividerAllSvg style={{ marginLeft: "28px" }} />
-      <ServicesLabel>All Resources</ServicesLabel>
-    </ServicesDividerRoot>
-  )
+export function PinnedResources(props: ResourceProps) {
+  return props.items?.length ? (
+    <React.Fragment>
+      <ServicesDividerRoot className={"ServicesDivider pinned"}>
```
Comment (nicks) [inline web/src/OverviewPane.tsx:50]: you should be able to get rid of these classes too now?

**Review by nicks — APPROVED** [review]

File: web/src/OverviewPane.tsx (hunk 3 above)
Comment (maiamcc) [reply web/src/OverviewPane.tsx:50]: whoops good catch

**Pull request merged**